### PR TITLE
Truncate too long surnames when synchronising users with ldap.

### PR DESCRIPTION
**Pull request by xliiv**
Before, ldap users with surname longer then 30 (Django's default) chars break
synching. Now such surnames are truncated to 30 chars.

### CODE REVIEW

File: src/ralph/account/management/commands/ldap_sync.py [hunk 1]
```diff
@@ -128,6 +143,8 @@ def populate_users(self):
         """Load users from ldap and populate them. Returns number of users."""
         synced = 0
         for user_dn, ldap_dict in self._get_users():
-            self._create_or_update_user(user_dn, ldap_dict)
             synced += 1
+            print(synced, user_dn, ldap_dict['sAMAccountName'])
```
Comment (ar4s) [inline src/ralph/account/management/commands/ldap_sync.py]: ?

File: src/ralph/account/management/commands/ldap_sync.py [hunk 2]
```diff
@@ -22,6 +23,20 @@
...
 
+def _truncate_surname(ldap_dict):
+    """
+    Truncate user's surname when it's longer then default django value, which is
+    30 chars.
+    """
+    if 'sn' in ldap_dict:
+        valid_sns = []
+        for surname in ldap_dict['sn']:
+            max_length = User._meta.get_field('last_name').max_length
```
Comment (zefciu) [inline src/ralph/account/management/commands/ldap_sync.py]: Why are you retrieving this in every iteration?

Maybe you could turn the whole loop into a comprehension?

File: src/ralph/account/management/commands/ldap_sync.py (hunk 1 above)
Comment (zefciu) [inline src/ralph/account/management/commands/ldap_sync.py]: ¿

Comment (zefciu) [inline src/ralph/account/management/commands/ldap_sync.py]: ؟

Comment (quamilek) [inline src/ralph/account/management/commands/ldap_sync.py]: :hamburger:

**Comment by zefciu** [comment]
:+1:

**Comment by ar4s** [comment]
:ok:

**Pull request merged**